packages/webapp/pages/squads/[handle]/moderate.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactElement, useEffect } from 'react';
+import {
+  ManageSquadPageContainer,
+  SquadSettingsProps,
+} from '@dailydotdev/shared/src/components/squads/utils';
+import {
+  SquadTab,
+  SquadTabs,
+} from '@dailydotdev/shared/src/components/squads/SquadTabs';
+import { SquadModerationList } from '@dailydotdev/shared/src/components/squads/moderation/SquadModerationList';
+import {
+  PageHeader,
+  PageHeaderTitle,
+} from '@dailydotdev/shared/src/components/layout/common';
+import {
+  Button,
+  ButtonVariant,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import { ArrowIcon } from '@dailydotdev/shared/src/components/icons';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import { useSquad } from '@dailydotdev/shared/src/hooks';
+import { useRouter } from 'next/router';
+import { getLayout as getMainLayout } from '../../../components/layouts/MainLayout';
+
+export default function ModerateSquadPage({
+  handle,
+}: SquadSettingsProps): ReactElement {
+  const { squad, isFetched } = useSquad({ handle });
+  const router = useRouter();
+
+  useEffect(() => {
+    if (!isFetched) {
+      return;
+    }
+
+    if (squad?.moderationRequired) {
+      return;
+    }
```

Comment:
This is intented? (not sure if it made more sense to show some copy to say it's disabled if your squad doesn't have this setting)

Reply:
maybe we want to split the logic:
- if admin -> Copy that push for going to settings and change them;
- if user -> redirect to squad page;

wdyt?

Reply:
@ilasw sounds good to me 🚀

Reply:
They won't have a way to go to this page if it is disabled, hence redirection works for me.

Reply:
Sounds good indeed.